fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -336,34 +330,10 @@ private fun ReviewQualityInfo(
         // follow-up when the copy is finalized.
         // Bug 1848219
         Text(
-            text = stringResource(id = R.string.review_quality_check_info_overview),
-            color = FirefoxTheme.colors.textPrimary,
-            style = FirefoxTheme.typography.body2,
-        )
-
-        val link = stringResource(R.string.review_quality_check_info_learn_more_link)
-        val text = stringResource(R.string.review_quality_check_info_learn_more, link)
-        val linkStartIndex = text.indexOf(link)
-        val context = LocalContext.current
-        val linkEndIndex = linkStartIndex + link.length
-        ClickableSubstringLink(
```

Comment:
Now this is removed, we can remove the `Suppress("Depecation")` annotation

Reply:
The `ClickableSubstringLink` was actually moved lower to match the updated design. We'll still need to suppress deprecation for that. I'll keep the current position and let the design inconsistencies to be fixed in 1848219.

Reply:
Sounds fair, thanks for the clarification!